cmd/celestia-appd/cmd/modify_root_command_multiplexer.go

```diff
@@ -36,18 +54,18 @@ func modifyRootCommand(rootCommand *cobra.Command) {
 		extraArgs = append(extraArgs, "--v2-upgrade-height="+v2UpgradeHeight)
 	}
 
-	versions, err := abci.NewVersions(abci.Version{
-		Appd:        appdV3,
-		ABCIVersion: abci.ABCIClientVersion1,
-		AppVersion:  3,
-		StartArgs: append([]string{
-			"--grpc.enable",
-			"--api.enable",
-			"--api.swagger=false",
-			"--with-tendermint=false",
-			"--transport=grpc",
-		}, extraArgs...),
-	})
+	versions, err := abci.NewVersions(
+		abci.Version{
+			Appd:        appdV3,
+			ABCIVersion: abci.ABCIClientVersion1,
+			AppVersion:  3,
+			StartArgs:   append(defaultArgs, extraArgs...),
+		}, abci.Version{
+			Appd:        appdV4,
+			ABCIVersion: abci.ABCIClientVersion1,
```

Comment:
[blocking] Pretty sure this should be ABCI version 2 but I haven't tried running this yet.

ABCI version 1 was used by v3
ABCI version 2 is used by v4

```suggestion
			ABCIVersion: abci.ABCIClientVersion2,
```

Reply:
Ah that makes sense